tests/models/test_horovod.py

```diff
@@ -147,28 +146,28 @@ def test_horovod_apex(tmpdir):
     )
     _run_horovod(trainer_options, on_gpu=True)
 
-
-@pytest.mark.skipif(platform.system() == "Windows", reason="Horovod is not supported on Windows")
-@pytest.mark.skipif(not _nccl_available(), reason="test requires Horovod with NCCL support")
-@pytest.mark.skipif(torch.cuda.device_count() < 2, reason="test requires multi-GPU machine")
-@pytest.mark.skipif(not NATIVE_AMP_AVALAIBLE, reason="test requires torch.cuda.amp")
-def test_horovod_amp(tmpdir):
-    """Test Horovod with multi-GPU support using native amp."""
-    trainer_options = dict(
-        default_root_dir=str(tmpdir),
-        weights_save_path=str(tmpdir),
-        gradient_clip_val=1.0,
-        # progress_bar_refresh_rate=0,
-        max_epochs=1,
-        limit_train_batches=0.4,
-        limit_val_batches=0.2,
-        gpus=2,
-        deterministic=True,
-        distributed_backend='horovod',
-        amp_backend='native',
-        # precision=16,
-    )
-    _run_horovod(trainer_options, on_gpu=True)
+# TODO: uncomment after horovod fixes with torch.cuda.amp
+# @pytest.mark.skipif(platform.system() == "Windows", reason="Horovod is not supported on Windows")
+# @pytest.mark.skipif(not _nccl_available(), reason="test requires Horovod with NCCL support")
+# @pytest.mark.skipif(torch.cuda.device_count() < 2, reason="test requires multi-GPU machine")
+# @pytest.mark.skipif(not NATIVE_AMP_AVALAIBLE, reason="test requires torch.cuda.amp")
+# def test_horovod_amp(tmpdir):
+#     """Test Horovod with multi-GPU support using native amp."""
+#     trainer_options = dict(
+#         default_root_dir=str(tmpdir),
+#         weights_save_path=str(tmpdir),
+#         gradient_clip_val=1.0,
+#         progress_bar_refresh_rate=0,
+#         max_epochs=1,
+#         limit_train_batches=0.4,
+#         limit_val_batches=0.2,
+#         gpus=2,
+#         deterministic=True,
+#         distributed_backend='horovod',
+#         amp_backend='native',
+#         precision=16,
+#     )
+#     _run_horovod(trainer_options, on_gpu=True)
```

Comment:
Why not add an additional pytest skip?
```suggestion
@pytest.mark.skip(reason="Skip till Horovod fixes integration with Native torch.cuda.amp")
@pytest.mark.skipif(platform.system() == "Windows", reason="Horovod is not supported on Windows")
@pytest.mark.skipif(not _nccl_available(), reason="test requires Horovod with NCCL support")
@pytest.mark.skipif(torch.cuda.device_count() < 2, reason="test requires multi-GPU machine")
@pytest.mark.skipif(not NATIVE_AMP_AVALAIBLE, reason="test requires torch.cuda.amp")
def test_horovod_amp(tmpdir):
    """Test Horovod with multi-GPU support using native amp."""
    trainer_options = dict(
        default_root_dir=str(tmpdir),
        weights_save_path=str(tmpdir),
        gradient_clip_val=1.0,
        progress_bar_refresh_rate=0,
        max_epochs=1,
        limit_train_batches=0.4,
        limit_val_batches=0.2,
        gpus=2,
        deterministic=True,
        distributed_backend='horovod',
        amp_backend='native',
        precision=16,
    )
    _run_horovod(trainer_options, on_gpu=True)
```

Reply:
Ohh yeah! I'm used to comment so...